web/src/app/alerts/CreateAlertDialog/StepContent/Step0.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Grid, TextField } from '@material-ui/core'
+import { FormField } from '../../../forms'
+
+export default () => (
```

Comment:
Give this and it's file a descriptive name instead of a number like `AlertDetailsStep` or something like that. Also use `function AlertDetailsStep` or assign `.displayName`